spatial/transform/umeyama_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package transform
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/mat"
+)
+
+func TestUmeyama(t *testing.T) {
+	tol := 1e-10
+
+	testCases := []struct {
+		name      string
+		src       *mat.Dense
+		dst       *mat.Dense
+		wantScale float64
+		wantRot   *mat.Dense
+		wantTrans *mat.VecDense
+		tolerance float64
+	}{
+		{
+			name: "2D_case_from_paper",
+			src: mat.NewDense(3, 2, []float64{
+				0, 0,
+				1, 0,
+				0, 2,
+			}),
+			dst: mat.NewDense(3, 2, []float64{
+				0, 0,
+				-1, 0,
+				0, 2,
+			}),
+			wantScale: 0.7211102550927978,
+			wantRot:   mat.NewDense(2, 2, []float64{0.8320502943378437, 0.554700196225229, -0.554700196225229, 0.8320502943378436}),
+			wantTrans: mat.NewVecDense(2, []float64{-0.7999999999999998, 0.4}),
```

Comment:
```suggestion
			wantRot:   mat.NewDense(2, 2, []float64{
				0.8320502943378437, 0.554700196225229,
				-0.554700196225229, 0.8320502943378436,
			}),
			wantTrans: mat.NewVecDense(2, []float64{
				-0.7999999999999998,
				0.4,
			}),
```
(similar below)

Can we ask for `[ -0.8; 0.4 ]` for the t with a reasonable tolerance? I would imagine so.

Reply:
We can. Ill adjust values like these for clarity